node/full_client_test.go

```diff
@@ -42,6 +42,16 @@ var expectedInfo = abci.ResponseInfo{
 
 var mockTxProcessingTime = 10 * time.Millisecond
 
+func getRandomValidatorSet() *tmtypes.ValidatorSet {
```

Comment:
Can we change the name of this to something like `getRandomSingleValidatorSet()` since this is a random set of one validator?